### PR TITLE
[REVIEW] Updating CMakeLists.txt to add aarch64 support

### PR DESCRIPTION
**Pull request by JasonAtNvidia**
Changing GPU_ARCHS portion of CMakeLists.txt to add flag to detect aarch64 architectures and chose GenCode flags based on those.

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Review by kkraus14 — APPROVED** [review]

**Comment by kkraus14** [comment]
ok to test

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: cpp/CMakeLists.txt [hunk 1]
```diff
@@ -89,15 +89,38 @@ if("${GPU_ARCHS}" STREQUAL "")
...
+  
+  # Check for embedded vs workstation architectures
+  if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
+    # This is being built for Linux4Tegra or SBSA ARM64
+    message(STATUS "Detected aarch64 cpu architecture: using L4T and SBSA compatible GPU_ARCHS")
+    set(GPU_ARCHS "62")
+    if((CUDA_VERSION_MAJOR EQUAL 9) OR (CUDA_VERSION_MAJOR GREATER 9))
+      set(GPU_ARCHS "${GPU_ARCHS};72")
+    endif()
+    if((CUDA_VERSION_MAJOR EQUAL 11) OR (CUDA_VERSION_MAJOR GREATER 11))
```
Comment (harrism) [inline cpp/CMakeLists.txt:101]: These should all be simplified using `GREATER_EQUAL`
```suggestion
    if(CUDA_VERSION_MAJOR GREATER_EQUAL 9)
      set(GPU_ARCHS "${GPU_ARCHS};72")
    endif()
    if(CUDA_VERSION_MAJOR GREATER_EQUAL 11)
```

File: cpp/CMakeLists.txt [hunk 2]
```diff
@@ -89,15 +89,38 @@ if("${GPU_ARCHS}" STREQUAL "")
...
+      set(GPU_ARCHS "${GPU_ARCHS};75;80")
+    endif()
+
+  else()
+    # This is being built for an x86 or x86_64 architecture
+    message(STATUS "Selecting GPU_ARCHS compatible with most GPUs")
+    set(GPU_ARCHS "60")
+    if((CUDA_VERSION_MAJOR EQUAL 9) OR (CUDA_VERSION_MAJOR GREATER 9))
```
Comment (harrism) [inline cpp/CMakeLists.txt:109]: GREATER_EQUAL on all of these too.

**Review by harrism — CHANGES_REQUESTED** [review]

**Comment by trxcllnt** [comment]
I believe this is handled in https://github.com/rapidsai/cuspatial/pull/365/

**Comment by JasonAtNvidia** [comment]
> I believe this is handled in #365

It does appear that the new upgrades to CMakeLists in #365 includes all of the features I added to other CMakeLists to add aarch64 support for Jetson.  I am going to close this bug based on that.

**Pull request closed without merging**